Orange/widgets/data/owpreprocess.py

```diff
@@ -637,55 +637,82 @@ def enum_to_index(enum, key):
+            self.layout().addWidget(rb)
 
-        form.addRow("Center:", self.__centercb)
-        form.addRow("Scale:", self.__scalecb)
-        self.layout().addLayout(form)
-        self.__centercb.currentIndexChanged.connect(self.changed)
-        self.__scalecb.currentIndexChanged.connect(self.changed)
-        self.__centercb.activated.connect(self.edited)
-        self.__scalecb.activated.connect(self.edited)
+    def setMethod(self, method):
+        b = self.__group.button(method)
+        if b is not None:
+            b.setChecked(True)
+            self.__method = method
+            self.changed.emit()
 
     def setParameters(self, params):
-        center = params.get("center", _Scale.CenteringType.Mean)
-        scale = params.get("scale", _Scale.ScalingType.Std)
-        self.__centercb.setCurrentIndex(
-            enum_to_index(_Scale.CenteringType, center))
-        self.__scalecb.setCurrentIndex(
-            enum_to_index(_Scale.ScalingType, scale))
+        method = params.get("method", Scale.NoCentering)
+        self.setMethod(method)
 
     def parameters(self):
-        return {"center": index_to_enum(_Scale.CenteringType,
-                                        self.__centercb.currentIndex()),
-                "scale": index_to_enum(_Scale.ScalingType,
-                                       self.__scalecb.currentIndex())}
+        return {"method": self.__method}
+
+    def __on_buttonClicked(self):
+        self.__method = self.__group.checkedId()
+        self.changed.emit()
+        self.edited.emit()
 
     @staticmethod
     def createinstance(params):
-        center = params.get("center", _Scale.CenteringType.Mean)
-        scale = params.get("scale", _Scale.ScalingType.Std)
-        return _Scale(center=center, scale=scale)
+        params = dict(params)
+        method = params.pop("method", Scale.CenterByMean)
+        if method == Scale.NoCentering:
+            center = params.get("center", _Scale.CenteringType.NoCentering)
+            scale = params.get("scale", _Scale.ScalingType.NoScaling)
+            return _Scale(center=center, scale=scale)
+        elif method == Scale.CenterByMean:
+            center = params.get("center", _Scale.CenteringType.Mean)
+            scale = params.get("scale", _Scale.ScalingType.NoScaling)
+            return _Scale(center=center, scale=scale)
+        elif method == Scale.CenterByMedian:
+            center = params.get("center", _Scale.CenteringType.Median)
+            scale = params.get("scale", _Scale.ScalingType.NoScaling)
+            return _Scale(center=center, scale=scale)
+        elif method == Scale.NormalizeSD:
+            return Normalize(norm_type=Normalize.NormalizeBySD)
+        elif method == Scale.NormalizeSpan:
+            return Normalize(norm_type=Normalize.NormalizeBySpan)
+        elif method == Scale.NormalizeSpan2:
+            return Normalize(norm_type=Normalize.NormalizeBySpan2, zero_based=False)
```

Comment:
`params` can no longer contain a key `"center"`, so `center = params.get("center", _Scale.CenteringType.NoCentering)` is equivalent to `center = _Scale.CenteringType.NoCentering`. Better

```
center = params.get("center", _Scale.CenteringType.Mean)
scale = params.get("scale", _Scale.ScalingType.NoScaling)
return _Scale(center=center, scale=scale)
```

is equivalent to 

```
return _Scale(_Scale.CenteringType.Mean, _Scale.ScalingType.NoScaling)
```

and so forth

(I would omit `center=` and `scale=` because the names of constants are descriptive enough).